src/app/components/utilities/richtext.scss

```diff
@@ -388,35 +384,23 @@
       }
 
       > h2 {
+        font-size: 1.5rem;
         font-family: var(--od-font-stack-serif);
+
+        @media #{$mq-gt-sm} {
+          font-size: 2rem;
+        }
       }
 
-      > h2,
       > hr {
         &::before {
           content: '';
           display: block;
-          margin: $unit * 3 auto $unit 0;
-          width: 10em;
+          margin: $unit * 2 auto $unit * 1 0;
+          width: 33%;
           height: $unit * 0.25;
           opacity: 0.4;
-          background-color: var(--nw-colour-surface-accent);
-
-          @media #{$mq-md} {
-            margin: $unit * 4 auto $unit * 1.5 0;
-          }
-
-          @media #{$mq-gt-md} {
-            margin: $unit * 5 auto $unit * 1.5 0;
-          }
-        }
-
-        @media #{$mq-md} {
-          padding-bottom: $unit * 0.5;
-        }
-
-        @media #{$mq-gt-md} {
-          padding-bottom: $unit;
+          background-color: var(--nw-colour-text-accent);
```

Comment:
We should make our own CSS var for this (which proxies a `--nw-` var if suitable).